CIP-0068/README.md

```diff
@@ -124,19 +124,23 @@ files_details =
   {
     ? name : bounded_bytes, ; UTF-8
     mediaType : bounded_bytes, ; UTF-8
-    src : bounded_bytes ; UTF-8
+    src : uri,
     ; ... Additional properties are allowed
   }
 
 metadata = 
   {
     name : bounded_bytes, ; UTF-8
-    image : bounded_bytes, ; UTF-8
-    ? mediaType : bounded_bytes, ; UTF-8
```

Comment:
(potentially requiring version bump to 2 or 3)

Reply:
Agreed!